### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ofir68681**

### CODE REVIEW

File: RavKavCard.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+class RavKavCard:
+
+    def __init__(self, id_num, profile, money):
+        self.id_num = id_num
+        self.profile = profile
+        self.money = money
+
+    def __str__(self):
+        return "id: " + str(self.id_num) + " profile: " + self.profile + " money must be funny: " + str(self.money)
```
Comment (benhorngh) [inline RavKavCard.py]: use f-string

File: RavKavCard.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import random
+
+
+class RavKavCard:
+
+    def __init__(self, id_num, profile, money):
```
Comment (benhorngh) [inline RavKavCard.py]: use typing

Comment (benhorngh) [reply RavKavCard.py]: Everywhere

File: RavKavCard.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        self.id_num = id_num
+        self.profile = profile
```
Comment (benhorngh) [inline RavKavCard.py]: use Enum for the profile

File: RavKavCard.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        self.id_num = random.randint(1, 999999999)
+        profiles = ["young", "student", "old"]
+        self.profile = random.choice(profiles)
+        self.money = random.randint(0, 1000)
+
+    def menu(self):
+        while True:
+            print("Select action: \n 1.Check balance. \n 2.Pay for a ride. \n 3.Deposit.")
+            choice = int(input("Enter your selection: "))
+            if choice == 1:
```
Comment (benhorngh) [inline RavKavCard.py]: move every action to a different function,
so it will look like:
`if choice == 1:
    action_1()`

File: RavKavCard.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    def check_money(self):
+        return self.money
+
+    def rand_card(self):
+        self.id_num = random.randint(1, 999999999)
+        profiles = ["young", "student", "old"]
+        self.profile = random.choice(profiles)
+        self.money = random.randint(0, 1000)
+
+    def menu(self):
```
Comment (benhorngh) [inline RavKavCard.py]: I think menu should be in different class/ function, not inside RavKavCard class.
RavKavCard suppose to represent ravkav card

File: RavKavCard.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    def pay(self, m):
+        if self.money >= m:
+            self.money -= m
+        else:
+            print("die")
+
+    def check_money(self):
+        return self.money
+
+    def rand_card(self):
```
Comment (benhorngh) [inline RavKavCard.py]: add documentation - what `rand_card` do

File: profile.py [hunk 7]
```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class Profile(Enum):
+    young = 1
```
Comment (benhorngh) [inline profile.py:6]: use upper case for enum keys,
and auto numbering instead of 1/2/3

File: RavKavCard.py (hunk 4 above)
Comment (ofir68681) [reply RavKavCard.py]: why should I do it? the code is organized and short for all the options.

Comment (benhorngh) [reply RavKavCard.py]: that way the menu function will be independent.
you will be able to change the logic without changing the menu itself

File: profile.py [hunk 8]
```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+# Enum class for RavKav profile
```
Comment (benhorngh) [inline profile.py:4]: the documentation should be inside the class, and with `""" docs """` instead of `# docs`

**Pull request merged**